tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
     "compilerOptions": {
+        "baseUrl": ".",
```

Comment:
Is this need to be edited by the user? If so, then tsconfig.json need to be included in `.gitignore` file I guess.

Reply:
or people that want to use a custom pricer will need to fork the repo in order to edit this file?

Reply:
well but then, other people will have trouble compiling the code because tsconfig.json file will never be found.

Reply:
no this just says, the project starts from this tsconfig location. it shouldn't need to be editted.

Reply:
I see. noted